lib/view_model/send/send_view_model.dart

```diff
@@ -225,6 +225,43 @@ abstract class SendViewModelBase extends WalletChangeListenerViewModel with Stor
     return wallet.balance[selectedCryptoCurrency]!.formattedFullAvailableBalance;
   }
 
+  @action
+  Future<void> updateSendingBalance() async {
+    // force the sendingBalance to recompute since unspent coins aren't observable
+    // or at least mobx can't detect the changes
+
+    final currentType = coinTypeToSpendFrom;
+
+    if (currentType == UnspentCoinType.any) {
+      coinTypeToSpendFrom = UnspentCoinType.nonMweb;
+    } else if (currentType == UnspentCoinType.nonMweb) {
+      coinTypeToSpendFrom = UnspentCoinType.any;
+    } else if (currentType == UnspentCoinType.mweb) {
+      coinTypeToSpendFrom = UnspentCoinType.nonMweb;
+    }
+
+    // set it back to the original value:
+    coinTypeToSpendFrom = currentType;
+  }
+
+  @computed
+  String get sendingBalance {
+    // only for electrum and monero wallets atm:
+    switch (wallet.type) {
+      case WalletType.bitcoin:
+      case WalletType.litecoin:
+      case WalletType.bitcoinCash:
+        return wallet.formatCryptoAmount(
+            unspentCoinsListViewModel.getSendingBalance(coinTypeToSpendFrom).toString());
+      case WalletType.monero:
+      case WalletType.wownero:
+        return wallet.formatCryptoAmount(
+            unspentCoinsListViewModel.getSendingBalance(coinTypeToSpendFrom).toString());
+      default:
+        return balance;
+    }
```

Comment:
also decred, and since they are the same function call, just make them all in the same case (i.e add monero, wow, decred to btc, ltc, bch)
```suggestion
    // only for electrum, monero, wownero, decred wallets atm:
    switch (wallet.type) {
      case WalletType.bitcoin:
      case WalletType.litecoin:
      case WalletType.bitcoinCash:
      case WalletType.monero:
      case WalletType.wownero:
      case WalletType.decred:
        return wallet.formatCryptoAmount(
            unspentCoinsListViewModel.getSendingBalance(coinTypeToSpendFrom).toString());
      default:
        return balance;
    }
```